lms/djangoapps/edxnotes/helpers.py

```diff
@@ -143,6 +143,7 @@ def preprocess_collection(user, course, collection):
                 u"text": sanitize_html(model["text"]),
                 u"quote": sanitize_html(model["quote"]),
                 u"updated": dateutil_parse(model["updated"]),
+                u"tags": [sanitize_html(tag) for tag in model["tags"]],
```

Comment:
This strips out dangerous HTML like script tags. The user can specify tags like `<strong>` or `<em>` though.